index.md

```diff
@@ -69,33 +81,39 @@ It is possible to run the Gradle process using a different JDK by passing in the
 
 To obtain the CLI it may be downloaded from Maven Central [here](https://repo1.maven.org/maven2/org/jboss/gm/cli).
 
-##### Disabling the alignment
+#### Troubleshooting
 
-You can disable GME _through the CLI_ using the `manipulation.disable` property:
+* Gradle build daemon disappeared unexpectedly
+   * This may happen in Gradle versions prior to 5.x if the shell environment contains non ASCII characters (e.g. the PROMPT symbol).
 
-	$ gradle -Dmanipulation.disable=true...
+### Alignment Configuration
 
-#### Applying the Plugin(s)
+#### Customising the Manipulation plugin version
```

Comment:
For consistency with other section headings, I guess you should capitalize **P**lugin **V**ersion here, as well as Disabling the **P**lugins later on.

Reply:
Thanks, changed.